agent-api/src/commonMain/kotlin/Model.kt

```diff
@@ -67,7 +68,9 @@ data class AgentSessionTestsPayload(val sessionId: String, val tests: List<Strin
 data class ExecClassData(
     val id: Long? = null,
     val className: String,
-    @Serializable(with = BitSetSerializer::class) val probes: Probes,
+    @Serializable(with = BitSetSerializer::class)
+    @Column("probes", PostgresType.BIT_VARYING)
```

Comment:
Little question. (I don't remember exactly but)
Does this option available? 
```suggestion
    @Column(ExecClassData::probes.name, PostgresType.BIT_VARYING)
```
Or name for annotation required at the compilation stage ?

Reply:
Unfortunately it won't work that way: annotations arguments must be a compile-time constants